docs/codeguide/accessibility.md

```diff
@@ -1,25 +1,90 @@
-# Accessibility
+# Accessibility (a11y)
+
+## Educational Resources / Training
+- [LinkedIn Learning - Accessibility for Web Design](https://www.linkedin.com/learning/accessibility-for-web-design/welcome?u=2168252)
+  - Real world examples
+  - Visual issues
+  - Navigating with keyboard
+  - Touch
+  - Images/video
+  - Forms
+- [A360 Docs](https://hub.accessible360.com/kb/articles) (must be logged in)
+- [a11y Project](https://www.a11yproject.com/)
 

```

Comment:
Can you please add a `---` divider just to make the flow of the document clearer. Thanks.